tools/clang/test/CodeGenSPIRV/constant.scalar.hlsl

```diff
@@ -61,4 +61,16 @@ void main() {
   double c_large  =  1234567898765.32;
 // CHECK-DAG: %double_n1234567898765_32 = OpConstant %double -1234567898765.32
   double c_nlarge = -1234567898765.32;
+
+// Since we translate HLSL 'half' to 32-bit float in SPIR-V, float constants are generated.
+// CHECK-DAG: %float_7_7 = OpConstant %float 7.7
```

Comment:
Hmm, I would suggest to convert all `CHECK-DAG` directives to `CHECK` directives in this file. `CHECK-DAG` takes a much longer time to run. The flexibility it gives us is not quite beneficial here.